css/properties/font-kerning.json

```
@@ -5,16 +5,27 @@
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/font-kerning",
           "support": {
-            "chrome": {
-              "prefix": "-webkit-",
-              "version_added": "32"
-            },
-            "chrome_android": {
-              "prefix": "-webkit-",
-              "version_added": "32"
-            },
+            "chrome": [
+              {
+                "version_added": "33"
+              },
+              {
+                "prefix": "-webkit-",
+                "version_added": "18",
```

29 according to caniuse

```suggestion
                "version_added": "29",
```